tests/testthat/helper-funs.R

```diff
@@ -0,0 +1,35 @@
+basicScatP <- function() {
+  ggplot2::ggplot(data = mtcars) +
+    ggplot2::geom_point(ggplot2::aes(x = wt, y = drat)) 
+}
+
+ggMarg2 <- function(type, ...) 
+  ggMarginal(p = basicScatP(), type = type, ...)
+
+funList <- 
+  list(
```

Comment:
Related to approach taken in test-ggMarginal, the assumption was that the unit tests that were written pretty much cover all the the expected behavior of `ggMarginal` (within reason) and additional tests would be added to the last `test_that` call as issues are uncovered and addressed. I don't think we will find many (if any) cases where a _feature_ is added to ggplot2 that impacts ggMarginal. The more likely situation will be where small gpar parameters change from ggplot version to version (i.e., colors related to the theme elements), but those will be covered by separating the test cases across ggplot2 versions. We'd also have to mix the `ggplot2Version` param in two levels of abstraction if we pass them into `funList`, which isn't so bad, but I would prefer to keep the idea of ggplot2 version in just one file if possible.